modules/integration/tests-common/integration-tests-utils/src/main/java/org/wso2/mb/integration/common/utils/ui/pages/main/TopicsBrowsePage.java

```diff
@@ -99,6 +99,10 @@ public TopicSubscribePage addSubscription(String topicName){
                 WebElement topicSubscribeLink = liNode.findElement(By.className("topicSubscribeStyle"));
                 topicSubscribeLink.click();
                 browsePage = new TopicSubscribePage(driver);
+                //We don't need to search through topics any more
+                //Continuing in the loop will cause an error when trying to find the next topic in the
+                //topic list since we have already browed the add subscription page
```

Comment:
Implementation of this method has changed and this part of the code is not present anymore.